src/viur/core/skeleton/meta.py

```diff
@@ -411,6 +411,8 @@ def fromClient(
                                 f"""{getattr(cls, "kindName", cls.__name__)}: {".".join(error.fieldPath)}: """
                                 f"""({error.severity}) {error.errorMessage}"""
                             )
+                    else:
+                        errors.clear()
```

Comment:
This is the only confusing line. It just ensures that bones which are not provided simply don't output any errors, according to the `amend`-rules. A bone with a concrete errors (invalid value submit) will always be reported.